addon/authenticators/jwt.js

```diff
@@ -44,6 +44,15 @@ export default BaseAuthenticator.extend({
   passwordAttributeName: 'password',
 
   /**
+    The name of the key in the response containing the JWT.
+    @property responseJwtName
+    @type String
+    @default 'token'
+    @public
+  */
+  responseJwtName: 'token',
```

Comment:
Can you remove _tokenAttributeName_ from the authorizer? It doesn't do anything there but it was supposed to do this

Reply:
Oh, I like that name better.  I was struggling to come up with a concise name for this property.  I'll add this option to the Authorizer too, since I'll probably be needing it in my app.